account_invoice_secondary_unit_price/models/account_invoice_line.py

```diff
@@ -37,3 +38,9 @@ def _onchange_uom_id(self):
         """
         super(AccountInvoiceLine, self)._onchange_uom_id()
         self.onchange_secondary_price()
+
+    @api.constrains("secondary_uom_id", "secondary_uom_price")
+    def _check_secondary_uom_id(self):
+        for line in self:
+            if (line.secondary_uom_price > 0) and (line.secondary_uom_id.id is False):
```

Comment:
```suggestion
            if not line.secondary_uom_id and line.secondary_uom_price != 0.0:
```
@kanda999 I think this should work.  Can you check it.

Reply:
It can work.